### PR TITLE
config: add Bridge and Bridge Management genesis allocations

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Bridge Proxy: 0x1212000000000000000000000000000000000004
Bridge Implementation: 0x1212100000000000000000000000000000000004
Bridge Management Proxy: 0x1212000000000000000000000000000000000005
Bridge Management Implementation: 0x1212100000000000000000000000000000000005

### CODE REVIEW

File: privnet/single/genesis_template.json [hunk 1]
```diff
@@ -68,6 +68,20 @@
         "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc": "0x1212100000000000000000000000000000000003"
       }
     },
+    "0x1212000000000000000000000000000000000004": {
+      "balance": "0",
```
Comment (mialbu) [inline privnet/single/genesis_template.json]: This needs the initial GAS allocation of Neo X. We have not implemented a funding feature (see https://github.com/bane-labs/bridge-evm-contracts/issues/74) yet. Until we do so (until T4), we need to provide the bridge proxy all the gas we'll need to distribute on Neo X.

**Review by mialbu — CHANGES_REQUESTED** [review]
Check my comment.

File: privnet/single/genesis_template.json (hunk 1 above)
Comment (AnnaShaleva) [reply privnet/single/genesis_template.json]: Well, what is the desired amount of GAS to distribute? @steven1227, what do you think?

File: config/genesis_testnet.json [hunk 2]
```diff
@@ -92,6 +106,41 @@
...
+      "storage": {
```
Comment (chenquanyu) [inline config/genesis_testnet.json]: `storage` should be in proxy contract.

File: privnet/single/genesis_template.json (hunk 1 above)
Comment (steven1227) [reply privnet/single/genesis_template.json]: 60,000,000 ?

File: config/genesis_testnet.json (hunk 2 above)
Comment (AnnaShaleva) [reply config/genesis_testnet.json]: @mialbu, agree? It's just the way how ERC1967Proxy works, but I need to ensure that everyone is OK with it.

File: config/genesis_testnet.json [hunk 3]
```diff
@@ -92,6 +106,41 @@
...
+        "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000001212000000000000000000000000000000000005",
+        "0x0000000000000000000000000000000000000000000000000000000000000001": "0x0000000000000000000000000000000000000000000000000000000000000000",
```
Comment (chenquanyu) [inline config/genesis_testnet.json]: Zero value doesn't need to be set in genesis file.

File: config/genesis_testnet.json (hunk 2 above)
Comment (mialbu) [reply config/genesis_testnet.json]: @AnnaShaleva Updated the PR on the bridge contracts: https://github.com/bane-labs/bridge-evm-contracts/pull/75/commits/3773a1c44eadeb66642fc1a7e63db9f4ece35535

File: config/genesis_testnet.json (hunk 3 above)
Comment (AnnaShaleva) [reply config/genesis_testnet.json]: They don't hurt anyway, so we may just keep them for the file consistency.

File: config/genesis_testnet.json (hunk 2 above)
Comment (AnnaShaleva) [reply config/genesis_testnet.json]: Fixed.

File: privnet/single/genesis_template.json (hunk 1 above)
Comment (AnnaShaleva) [reply privnet/single/genesis_template.json]: Fixed.

**Review by chenquanyu — APPROVED** [review]

**Review by mialbu — APPROVED** [review]

**Pull request merged**